src/transformation/visitors/call.ts

```diff
@@ -135,12 +136,16 @@ export function transformContextualCallExpression(
     ) {
         // table:name()
         const table = context.transformExpression(left.expression);
-        return lua.createMethodCallExpression(
-            table,
-            lua.createIdentifier(left.name.text, left.name),
-            transformedArguments,
-            node
-        );
+        let name: string = left.name.text;
```

Comment:
```suggestion
        let name = left.name.text;
```
slightly surprised our linter doesnt complain here